Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -6327,58 +6336,67 @@ ElementInfo IXPath.UseRootElement()
         }
 
         ElementInfo IXPath.GetElementParent(ElementInfo ElementInfo)
-        {            
-            object parentElement;
+        {
+            HtmlNode parentElementHtmlNode;
+            IWebElement parentElementIWebElement;
             object childElement = ElementInfo.ElementObject;
 
-            if(childElement == null)
-                Driver.FindElement(By.XPath(ElementInfo.XPath));
+            parentElementHtmlNode = ((HtmlNode)childElement).ParentNode;
+            parentElementIWebElement = ((IWebElement)childElement).FindElement(By.XPath(".."));
 
```

Comment:
Fix  parentElementHtmlNode = ((HtmlNode)childElement).ParentNode; to work correctly with "HTMLElementObject"
need to check if should do only one operation instead always both.

check if need to include the ElementInfo parentEI = xxx to HTMLElementObject is null checks